File: Apex Legends Cheat.cpp
#include <QApplication>
#include <QCheckBox>
#include <QPushButton>
#include <QLabel>
#include <QVBoxLayout>
#include <QGroupBox>
#include <QWidget>
#include <QStringList>

class SentryPanel : public QWidget {
    Q_OBJECT

public:
    SentryPanel(QWidget *parent = nullptr) : QWidget(parent) {
        setWindowTitle("Sentry - BuckCheat Trial");
        setFixedSize(420, 400);

        // Branding
        QLabel *title = new QLabel("🛡️ Sentry by BuckCheat");
        title->setStyleSheet("font-weight: bold; font-size: 20px; color: #006400;");
        title->setAlignment(Qt::AlignCenter);

        QLabel *trial = new QLabel("Trial Mode: 7 Days Remaining");
        trial->setStyleSheet("color: gray;");
        trial->setAlignment(Qt::AlignCenter);

        statusLabel = new QLabel("🛑 System Idle");
        statusLabel->setStyleSheet("color: darkred; font-size: 14px;");
        statusLabel->setAlignment(Qt::AlignCenter);

        // Features
        QGroupBox *featureBox = new QGroupBox("Sentry Modules");
        QVBoxLayout *featureLayout = new QVBoxLayout;

        aimbot = new QCheckBox("Auto Aim");
        wallscan = new QCheckBox("Wall Scan");
        norecoil = new QCheckBox("Zero Recoil");
        radar = new QCheckBox("Mini Radar");

        featureLayout->addWidget(aimbot);
        featureLayout->addWidget(wallscan);
        featureLayout->addWidget(norecoil);
        featureLayout->addWidget(radar);
        featureBox->setLayout(featureLayout);

        connect(aimbot, &QCheckBox::stateChanged, this, &SentryPanel::updateStatus);
        connect(wallscan, &QCheckBox::stateChanged, this, &SentryPanel::updateStatus);
        connect(norecoil, &QCheckBox::stateChanged, this, &SentryPanel::updateStatus);
        connect(radar, &QCheckBox::stateChanged, this, &SentryPanel::updateStatus);

        // Start Button
        QPushButton *startBtn = new QPushButton("🚀 Launch Sentry");
        startBtn->setStyleSheet("background-color: green; color: white; font-size: 14px;");
        connect(startBtn, &QPushButton::clicked, this, &SentryPanel::startSentry);

        // Main Layout
        QVBoxLayout *mainLayout = new QVBoxLayout;
        mainLayout->addWidget(title);
        mainLayout->addWidget(trial);
        mainLayout->addSpacing(10);
        mainLayout->addWidget(statusLabel);
        mainLayout->addSpacing(10);
        mainLayout->addWidget(featureBox);
        mainLayout->addSpacing(10);
        mainLayout->addWidget(startBtn);

        setLayout(mainLayout);
    }

private slots:
    void updateStatus() {
        QStringList active;

        if (aimbot->isChecked()) active << "Auto Aim";
        if (wallscan->isChecked()) active << "Wall Scan";
        if (norecoil->isChecked()) active << "Zero Recoil";
        if (radar->isChecked()) active << "Mini Radar";

        if (!active.isEmpty()) {
            statusLabel->setText("✅ Active Modules: " + active.join(", "));
            statusLabel->setStyleSheet("color: green; font-size: 14px;");
        } else {
            statusLabel->setText("🛑 System Idle");
            statusLabel->setStyleSheet("color: darkred; font-size: 14px;");
        }
    }

    void startSentry() {
        if (aimbot->isChecked() || wallscan->isChecked() || norecoil->isChecked() || radar->isChecked()) {
            statusLabel->setText("✅ Sentry System Engaged (Demo)");
            statusLabel->setStyleSheet("color: green; font-size: 14px;");
        } else {
            statusLabel->setText("⚠️ Enable at least one module first");
            statusLabel->setStyleSheet("color: orange; font-size: 14px;");
        }
    }

private:
    QLabel *statusLabel;
    QCheckBox *aimbot;
    QCheckBox *wallscan;
    QCheckBox *norecoil;
    QCheckBox *radar;
};

#include "sentry_panel.moc"

int main(int argc, char *argv[]) {
    QApplication app(argc, argv);
    SentryPanel panel;
    panel.show();
    return app.exec();
}

File: Apex Legends Cheat.py
import tkinter as tk

class ToggleModule(tk.Frame):
    def __init__(self, parent, label_text, controller):
        super().__init__(parent)
        self.var = tk.IntVar()
        self.chk = tk.Checkbutton(self, text=label_text, variable=self.var, command=controller)
        self.chk.pack(anchor="w")

    def is_enabled(self):
        return bool(self.var.get())

    def label(self):
        return self.chk.cget("text")


class SentryControlPanel(tk.Tk):
    def __init__(self):
        super().__init__()
        self.title("Sentry Control - BuckCheat Trial")
        self.geometry("440x400")
        self.resizable(False, False)

        self.modules = []
        self._init_layout()

    def _init_layout(self):
        # Top Branding
        branding = tk.Label(self, text="Sentry by BuckCheat", font=("Courier", 18, "bold"), fg="#004d00")
        branding.pack(pady=10)

        trial_notice = tk.Label(self, text="Trial Use: Valid for 7 Days", fg="#666666")
        trial_notice.pack()

        # Status Label
        self.status_label = tk.Label(self, text="System Idle", font=("Arial", 12), fg="darkred")
        self.status_label.pack(pady=15)

        # Modules
        module_frame = tk.Frame(self)
        module_frame.pack(pady=10, padx=20, fill="x")

        features = ["Auto Aim", "Wall Scan", "Zero Recoil", "Mini Radar"]
        for label in features:
            mod = ToggleModule(module_frame, label, self.refresh_status)
            mod.pack(anchor="w", pady=3)
            self.modules.append(mod)

        # Start Button
        btn = tk.Button(self, text="Start Sentry System", bg="#008000", fg="white", command=self.start_sentry)
        btn.pack(pady=20, ipadx=8, ipady=4)

    def refresh_status(self):
        enabled = [mod.label() for mod in self.modules if mod.is_enabled()]
        if enabled:
            msg = "Modules Active: " + ", ".join(enabled)
            self.status_label.config(text=msg, fg="green")
        else:
            self.status_label.config(text="System Idle", fg="darkred")

    def start_sentry(self):
        if any(mod.is_enabled() for mod in self.modules):
            self.status_label.config(text="Sentry System Engaged (Simulation)", fg="green")
        else:
            self.status_label.config(text="Enable at least one module first", fg="orange")


if __name__ == "__main__":
    SentryControlPanel().mainloop()